ledger-core/virtual/integration/method_pulse_change_test.go

```diff
@@ -517,15 +514,7 @@ func TestVirtual_MethodCall_IfConstructorIsPending(t *testing.T) {
 					assert.Empty(t, res.DelegationSpec)
 					return false
 				})
-				typedChecker.VDelegatedRequestFinished.Set(func(res *payload.VDelegatedRequestFinished) bool {
-					getDelegated = true
-					require.Equal(t, delegationToken, res.DelegationSpec)
-					return false
-				})
-				typedChecker.VDelegatedCallResponse.Set(func(res *payload.VDelegatedCallResponse) bool {
-					delegationToken = res.ResponseDelegationSpec
-					return false
-				})
+				typedChecker.VDelegatedCallResponse.SetResend(false)
```

Comment:
then add check VDelegatedCallResponse count before mc.Finish()

Reply:
why? we test logic around delegations a lot. I think that in this scenario it's most important to check memory in executor mock and message with result of execution. That's it and we do both.